app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+class InvalidCommandFormat(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command = command.split()
+
+    if command[0] != "cp":
```

Comment:
it's better to unpack values into descriptive variables, for example command, old_file, copy_file = command.split()